MVC/models/comments.model.js

```diff
@@ -0,0 +1,30 @@
+const db = require("../../db/connection.js");
+const format = require("pg-format");
+
+const nonExistentCommentCustomError = {
+  status: 410,
+  message: "??? There isn't a comment stored in this parameter.",
+};
+
+const fetchComments = () => {
+  return db.query(`SELECT * FROM comments;`);
+};
+
+const fetchSpecificComment = (comment_id) => {
+  return db
+    .query(format(`SELECT * FROM comments WHERE comment_id = %L`, [comment_id]))
+    .then((result) => {
+      if (result.rows.length === 0) {
+        return Promise.reject(nonExistentCommentCustomError);
+      }
+      return result;
+    });
+};
+
+const removeSpecificComment = (comment_id) => {
+  return fetchSpecificComment(comment_id).then((result) => {
+    return db.query(`DELETE FROM comments WHERE comment_id = $1`, [comment_id]);
+  });
```

Comment:
You don't need to invoke `fetchSpecificComment` in order to delete it. You can handle it directly by deleting the comment using its comment_id 💡

Reply:
I know but I needed to throw an error if the comment doesn't exist in the first place, or is that unnecessary